hail_search/queries/base.py

```diff
@@ -820,7 +922,7 @@ def _filter_by_annotations(self, ht, is_comp_het=False, consequence_ids=None, an
             self._has_allowed_transcript_filter(ht, field) if field in transcript_filter_fields else ht[field]
             for field in filter_fields
         ]
-        return ht.filter(hl.any(all_filters))
+        return ht if not all_filters else ht.filter(hl.any(all_filters))
```

Comment:
I don't think there is one, I must've added this when a test was failing but that points to my code being wrong so I'm undoing this change.

Reply:
okay, the reason I added this because I am seeing this error during my recessive test case:

TypeError: any: parameter 'collection': expected (expression of type set<any> or array<any> or expression of type bool), found list: []

And it's not returning early because is_comp_het is true.

Reply:
yes,`all_filters ` should always be populated at this point as if this is not a compound het search we should already have returned (line 804 in master), and if it is a compound het search there have to be consequence filters applied (you can't even submit a comp het search without annotation filters)

Reply:
ah. my issues was that I created a recessive test case without including filters, so I updated my test.